utils/responseGenerator/generate_responses.ts

```diff
@@ -91,6 +91,9 @@ const main = async () => {
     );
 
     console.log(`Getting form template from ${appEnv}`);
+    if (appEnv === "staging") {
+      process.env.AWS_PROFILE = "staging";
```

Comment:
@bryan-robitaille 

I think you originally added -- this 

Can you review to make sure we still need this?